recipes/diffusioncma/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "diffusioncma" %}
+{% set version = "0.5.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/Olllom/diffusioncma/-/archive/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ba9bb31345bfcfc622a4caa675fb2e9d2c3db0c783beb0f1afd877ef2cf02c00
+
+build:
+  number: 0
+  skip: True  # [win or py<35]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake >=3.12.1 
+    - git
+    - libgomp  # [not osx]
+    - autoconf  # [unix]
+    - automake  # [unix]
+    - libtool  # [unix]
+    - llvm-openmp  # [osx]
+
+  host:
+    - python {{ python }}
```

Comment:
`{{ python }}` should not be required here.

Reply:
Good point. Thank you.